webui/react/src/pages/FlatRuns/FlatRuns.settings.ts

```diff
@@ -0,0 +1,81 @@
+import * as t from 'io-ts';
+
+import { INIT_FORMSET } from 'components/FilterForm/components/FilterFormStore';
+import { SettingsConfig } from 'hooks/useSettings';
+import { DEFAULT_SELECTION, SelectionType } from 'pages/F_ExpList/F_ExperimentList.settings';
+
+import { defaultColumnWidths, defaultRunColumns } from './columns';
+
+// have to intersect with an empty object bc of settings store type issue
+export const FlatRunsSettings = t.intersection([
+  t.type({}),
+  t.type({
```

Comment:
we only needed to do the intersection because the second object was all partial -- if all the properties of the object are required, then the workaround isn't necessary.